smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -48,10 +48,11 @@ contract MixinTransfer is
   function shareKey(
     address _to,
     uint _tokenId,
-    uint _timeShared
+    uint _timeShared,
+    address _keyManager
```

Comment:
Do we really want this? I think we can simplify and exclude keyManager here.

Thin keyManager does not change in this scenario. And no keyManager means that the keyOwner has those rights.

Reply:
Yeah we can simplify this. As we don't have a getter function for keyManager anymore, it probably makes the most sense to say that for all functions:
- if `address(0)` is passed as the keyManager, then the key owner is also the manager.(ie: `grantKeys`)
- if there's no param for keyManager in a function, then the key owner is also the manager.(ie: `shareKey`)

Reply:
>Thin keyManager does not change in this scenario. And no keyManager means that the keyOwner has those rights.

I just want to take an extra look at this. in this scenario, the existing keyManager is the one sharing the key. would you like to see the keyManager of the "shared" key here be the existing keyManager(of the parent key) or the keyOwner(of the child key)?